### PR TITLE
Minor fix for python2.7 and adding a should_copy_script argument

### PR DESCRIPTION
**Pull request by azjps**
(I checked that following works on python 2.7.)

In #47 a work around for a SGE-related issue was introduced,
where the worker script was copied to the current working
directory. The should_copy_script argument allows toggling
this behavior (defaulted to True so that SGE users do not
need to explicitly specify the fix).

Also fixed a minor python2 backwards-compatibility issue,
where `shutil.copy()` returns `None`.

### CODE REVIEW

File: dask_drmaa/core.py [hunk 1]
```diff
@@ -75,6 +76,9 @@ def __init__(self, template=None, cleanup_interval=1000, hostname=None,
         script: string (optional)
             Path to the dask-worker executable script.
             A temporary file will be made if none is provided (recommended)
+        should_copy_script: bool
```
Comment (mrocklin) [inline dask_drmaa/core.py]: For brevity how about `copy_script=True`

**Comment by mrocklin** [comment]
+1 from me.

**Comment by jakirkham** [comment]
Seems fine to me. @mrocklin thoughts?

**Comment by azjps** [comment]
Updated, not sure why Travis is failing.

**Comment by jakirkham** [comment]
Python 3.6 is failing due to a known flaky test. ( https://github.com/dask/dask-drmaa/issues/41 ) The Python 2.7 failure occurs on the same flaky test, but with a slight twist on the error. Could you please do a force push? I'd like to save the Travis CI log to revisit later.

**Comment by jakirkham** [comment]
Thanks @azjps.

**Pull request merged**